modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/PolicyDetail/index.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @liferay/empty-line-between-elements */
+/* eslint-disable @liferay/no-abbreviations */
```

Comment:
Is this really necessary?

Reply:
Yes. I need to concatenate a space between two strings.